server.js

```diff
@@ -47,11 +49,9 @@ app.put('/products/update', (req, res) => {
   })
 })
 
-app.delete('/products/delete', (req, res) => {
```

Comment:
Nice job editing that end point.  The verb in the request lets you know that it's a delete operation.